### PR TITLE
[Analytics] Adds how to capture GraphQL queries with Chrome DevTools

### PR DESCRIPTION
**Pull request by angelampcosta**

### CODE REVIEW

**Review by marciocloudflare — APPROVED** [review]

File: content/analytics/graphql-api/tutorials/capture-graphql-queries-from-dashboard/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+pcx_content_type: how-to
+title: Capture GraphQL queries with Chrome DevTools
+---
+
+# Capture GraphQL queries with Chrome DevTools
+
```
Comment (chris-martinelli) [inline content/analytics/graphql-api/tutorials/capture-graphql-queries-from-dashboard/index.md:7]: Using [Chrome DevTools](https://developer.chrome.com/docs/devtools) you can capture the queries running behind the Cloudflare Dashboard analytics. In this example, we'll focus on the Network Analytics dataset but the same process can be applied to any of the analytics available in your dashboard.

File: content/analytics/graphql-api/tutorials/capture-graphql-queries-from-dashboard/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account.
+2. Go to **Analytics & Logs** and select the **Network Analytics** dashboard or any other analytics dashboard you are interested in seeing the GraphQL queries in.
+
+![Analytics tab](/images/analytics/analytics-tab.png)
+
+3. Open the [Chrome Developer Tools](https://developer.chrome.com/docs/devtools) and select **Inspect**.
```
Comment (chris-martinelli) [inline content/analytics/graphql-api/tutorials/capture-graphql-queries-from-dashboard/index.md:15]: Screenshot 2024-07-31 at 10 27 08

File: content/analytics/graphql-api/tutorials/capture-graphql-queries-from-dashboard/index.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+![Type graphql in the search field](/images/analytics/search-field.png)
+
+6. Select one of the GraphQL requests to open its details and go to the **Payload** tab. There you will find the GraphQL query. Select the query line and then **Copy value** to capture the query.
+
+![Copy query value](/images/analytics/copy-value.png)
+
+7. If you want to capture a new query, adjust the filters in the **Network analytics** dashboard and a new query will appear in the GraphQL requests.
+
+![Create a new query](/images/analytics/new-query.png)
```
Comment (chris-martinelli) [inline content/analytics/graphql-api/tutorials/capture-graphql-queries-from-dashboard/index.md]: You can now use this query as the basis for your API call. See [getting started](https://developers.cloudflare.com/analytics/graphql-api/getting-started/)

**Comment by chris-martinelli** [comment]
Can we replace [assets/images/analytics/analytics-tab.png](https://github.com/cloudflare/cloudflare-docs/pull/15907/files#diff-9f3766c9af62240c0b990af0a7af04bdb4b23eab6eb362806f33f4a9b0e430b5) with this new image?
Screenshot 2024-07-31 at 10 22 30

**Review by chris-martinelli — COMMENTED** [review]
small clarifications to beginning and end of document

**Pull request merged**